### PR TITLE
Add create role to interactive cli

### PR DESCRIPTION
**Pull request by grimadas**
Add new permission model to interactive cli

### CODE REVIEW

**Comment by neewy** [comment]
```
The following tests FAILED:
	  3 - ametsuchi_test (Failed)
	  4 - wsv_query_command_test (Failed)
```
Can you check it?

**Comment by kamilsa** [comment]
https://github.com/hyperledger/iroha/pull/602#issuecomment-333799218 those tests also fail on `feature/permission` branch

File: irohad/ametsuchi/impl/storage_impl.hpp [hunk 1]
```diff
@@ -122,9 +122,13 @@ namespace iroha {
...
      protected:
       const std::string init_ = R"(
+CREATE TABLE IF NOT EXISTS role (
+    role_id character varying(45),
+    PRIMARY KEY (role_id)
+);
 CREATE TABLE IF NOT EXISTS domain (
     domain_id character varying(164),
-    default_role character varying(164),
+    default_role character varying(45) NOT NULL REFERENCES role(role_id),
```
Comment (neewy) [inline irohad/ametsuchi/impl/storage_impl.hpp]: that is good! I had this question in validation process, so I guess you can delete this task from the backlog

File: irohad/model/generators/impl/transaction_generator.cpp [hunk 2]
```diff
@@ -31,7 +31,6 @@ namespace iroha {
         tx.created_ts = timestamp;
         tx.creator_account_id = "";
         tx.tx_counter = 0;
-
         CommandGenerator command_generator;
         // Add peers
         for (size_t i = 0; i < peers_address.size(); ++i) {
```
Comment (neewy) [inline irohad/model/generators/impl/transaction_generator.cpp]: Why this todo is without a JIRA task?

**Pull request merged**